docs/sources/installation/ubuntulinux.md

```diff
@@ -169,6 +169,9 @@ To create the `docker` group and add your user:
 
 		$ docker run hello-world
 
+	If this fails with a message similar to below, check that the `DOCKER_HOST` env var is not set for your shell.
```

Comment:
Thank you for the contribution.  Maybe go ahead and give them the recover. I'm just guessing it is this:

--> https://gist.github.com/moxiegirl/13a9870072a2ef4be15b

If this fails with a message similar this:

		Cannot connect to the Docker daemon. Is 'docker -d' running on this host?

Check that the `DOCKER_HOST` environ variable is not set for your shell. If it is, unset it.